src/xcode/ENA/ENA/Source/Client/Security/__tests__/HTTPClientCertificatePinningTests.swift

```diff
@@ -51,4 +52,50 @@ class HTTPClientCertificatePinningTests: CWATestCase {
 		waitForExpectations(timeout: 30)
     }
 
+	/// Testing certificate pinning in the main endpoints on `production` and `wru`
+	func testAllProductionEndpoints() throws {
+		let environments = [EnvironmentDescriptor.production, EnvironmentDescriptor.custom("wru")]
+		environments.forEach { descriptor in
+			let env = Environments().environment(descriptor)
+			let hosts = [
+				env.dataDonationURL,
+				env.distributionURL,
+				env.errorLogSubmissionURL,
+				env.submissionURL,
+				env.verificationURL
+				// TODO: add certificate host URL //swiftlint:disable:this todo
```

Comment:
ToDo with swiftlint disable, why not use a simple comment here?

Reply:
I'll check for TODOs locally. In the past someone came up with splitting them to "TO DO" with the result of missing out tons of stuff and producing issues all over the place.

Reply:
What about calling it "TODOdubidubidu"? 🤪

Reply:
I'll leave it in as long as I struggle to get [Kai's PR](https://github.com/corona-warn-app/cwa-app-ios/pull/2835) through our beloved CI.